.github/workflows/release.yml

```diff
@@ -1,39 +1,42 @@
-name: .NET Core
+name: Create Chat Sample Release
 
 on:
   release:
     types: [created]
-    name: Release
+    name: Release  
 
 jobs:
   release:
     name: Release
-    runs-on: windows-latest
+    runs-on: ubuntu-latest
     steps:
     
       - name: Checkout
         uses: actions/checkout@v1
 
-      - name: Setup dotnet
-        uses: actions/setup-dotnet@v1
+      - name: Setup Node 14
+        uses: actions/setup-node@v2
         with:
-          dotnet-version: 3.1.101
+          node-version: '14'
+
       - name: Build
         shell: bash
         run: |
           release_name=Group-Chat
           unset CI
           # Build everything
-          dotnet publish ./Chat -c Release -o "$release_name"
+          npm run setup
+          npm run build
+          npm run package
 
-          7z a -tzip "group-chat.zip" "./${release_name}/*"
+          7z a -tzip -r "group-chat.zip" ./Chat/dist/*
 
           # Delete output directory
-          rm -r "$release_name"
+          # rm -r "./Chat/dist"
```

Comment:
remove if not used

Reply:
Going to uncomment, We were deleting the build artifacts before so we should continue to do this